Makefile

```diff
@@ -0,0 +1,52 @@
+BIN     = $$(npm bin)
+BABEL   = $(BIN)/babel
+KARMA   = $(BIN)/karma
+WEBPACK = $(BIN)/webpack
+
+.PHONY: clean test test-watch build package.json javascript release example javascript-min
+
+build:
+	@make clean
+	@make package.json
+	@make documentation
+	@make javascript
+	@make audit
+
+javascript-min: dist/src/Microcosm.js
+	@$(WEBPACK) -p $^ dist/microcosm.build.js \
+	--devtool sourcemap --output-library-target commonjs2
+
+javascript: $(shell find src -name '*.js' ! -path '*/__tests__/*')
+	@mkdir -p dist
+	@$(BABEL) -d dist $^
+	@make javascript-min
+
```

Comment:
`$^` in here says "Take the list of files described on line 19 and spread them out here"